packages/components/src/interfaces/internal/Askgov.ts

```diff
@@ -0,0 +1,10 @@
+import type { IsomerSiteProps } from "~/types"
+
+export interface AskgovProps {
+  "data-agency": string
+  "data-topic"?: string
```

Comment:
[question]: what happens if `data-topic` is left empty or omitted? (ie, if i put `""` or if it's `undefined`)

Reply:
Oh most cases it will be empty and not used. This is only for niche cases where agencies wanna pre-filter and only show questions from certain topics